### DISCUSSION TITLE
pobierz.pl

### DISCUSSION THREAD

**Discussion by blocker999** [FiltersHeroes/KAD#2347] [Pomysły]
### Dodatkowe informacje mogące mieć znaczenie

![p](https://github.com/FiltersHeroes/KAD/assets/23176832/41da7109-2af1-4068-b2f2-c10d6096ca3b)

**Comment by krystian3w** [comment]
Przykładowo 7-zip ma poprawny link do anglojęzycznego serwera.

![obraz](https://github.com/FiltersHeroes/KAD/assets/35370833/c9461c3c-b4c7-4f39-af56-745ce4d79fc9)


Wszystkich WordPressów, gdzie admin zaginął raczej nie da się wyłapać.

**Comment by hawkeye116477** [comment]
Na VT żaden AV nic nie wyłapuje. Pewnie kliknął w jakiś link z asystentem pobierania lub czymś podobnym.

A tu raport z HA => https://www.hybrid-analysis.com/sample/999e0c28f25da13c41b55891ccd65668788a57069cb863dc586c1c20ad1ac95e

**Reply by krystian3w** [reply]
Lub Comodo zbanował cały serwer i ma gdzieś prośby o usunięcie z ich czarnej listy.
